go.mod

```
@@ -1,6 +1,6 @@
 module github.com/elastic/stream
 
-go 1.19
+go 1.21
```

```suggestion
go 1.20
```
to agree with claims.